### DISCUSSION TITLE
Change the References heading to another language

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#777] [Q&A] [answered]
Is there a way to change the heading of References to, for example, Referencias (Spanish)?

**Comment by jjallaire** [comment, marked as answer]
You can change all of our UI text to Spanish by adding this metadata to your document:

```
lang: es
```

See https://quarto.org/docs/authoring/language.html

**Reply by ManuelSpinola** [reply]
Thank you very much.
